template-processors/applier/README.md

```diff
@@ -0,0 +1,13 @@
+# OpenShift Applier Template Processor
+
+This image allows for the use of [OpenShift Applier](https://github.com/redhat-cop/openshift-applier.git) with [Eunomia](https://github.com/KohlsTechnology/eunomia.git), a GitOps Operator for Kubernetes.
+
+This image can be pulled from quay.io with:
+
+    docker pull quay.io/redhat-cop/eunomia-applier
```

Comment:
Would we want to put this into our main org?

Reply:
Yes, we would. I'll change that.

Reply:
Done.